CONTRIBUTING.md

```diff
@@ -169,3 +169,34 @@ Support branches? [support/]
 Version tag prefix? [] eXist-
 Hooks and filters directory? [.git/hooks]
 ```
+
+
+## Commit Naming
+Commits to eXist-db by developers *must* follow a naming convention. This convention expects a prefix to be prepended to each of your commit messages. The prefixes enable us to better understand the commit and also to generate Change Logs. The prefixes in use are:  
+
+* `[feature]`
+    This should be used when a commit adds a new feature.
+* `[bugfix]`
```

Comment:
this should be `[fix]` for brevity

Reply:
No It should not. We have been using these naming conventions for years, lets not start changing them now. We have scripts that parse these!

Reply:
we have [bugfix] and [hotfix] ... is [hotfix] now dropped?

Reply:
I do see `[fix]` is already in use too.

Reply:
I even found `[documentationfix]` in there ...

Reply:
> I do see [fix] is already in use too.

Looking at `develop` branch only - this appears in only 71 commits which is less than 0.2% of all commits. Out of those 71 commits that include `[fix]` in the commit subject, more than 50% of them were from yourself @line-o!
The others are accounted for by just one or two occurrences from other frequent committers to the project who have clearly in that instance accidentally used `[fix]` instead of `[bugfix]`.

Reply:
> I even found [documentationfix] in there ...

```
❯  git log --pretty=format:"%an %s" | grep "\[documentationfix\]"
```

There is only 1 commit in the `develop` branch with that message. It is from an old core contributor to eXist-db, whom otherwise it can be seen clearly used `[bugfix]`